src/main/scala/strawman/collection/mutable/FlatHashTable.scala

```diff
@@ -0,0 +1,426 @@
+package strawman
```

Comment:
I essentially copy-pasted this file from the current collections. I just adapted the procedure syntax to make it dotty compatible and made `final` the methods that were annotated with `@deprecatedOverriding`.